tensorflow/compiler/xla/service/gpu/cudnn_conv_rewriter.cc

```diff
@@ -494,43 +494,55 @@ CudnnConvBackendConfig GetDefaultBackendConfig() {
   return config;
 }
 
-// Tries to rewrite a single convolution into a call to cudnn.
-StatusOr<bool> RunOnInstruction(HloInstruction* conv) {
-  CHECK_EQ(conv->opcode(), HloOpcode::kConvolution);
+// Helper function to create a custom_call instruction to replace the given
+// conv instruction
+static StatusOr<HloInstruction*> CreateCustomCall(HloInstruction* conv) {
```

Comment:
Pull the lambda out to make a regular helper function to it with TF_ASSIGN_OR_RETURN, so that error status will return to the caller.